code/renderers/react/template/stories/docgen-components/jsdoc/argTypes.snapshot

```diff
@@ -101,20 +101,10 @@
     "name": "case13",
     "table": {
       "defaultValue": null,
-      "jsDocTags": {
```

Comment:
this one has disappeared because the source has no name:

```ts
  /**
   * param with type
   * @param {SyntheticEvent}
   */
  case13: PropTypes.func,
```

in the old source, this code looks like it should've caught that and returned `null`:

```ts
  const paramName = tag.name;

  // When the @param doesn't have a name but have a type and a description, "null-null" is returned.
  if (paramName != null && paramName !== 'null-null') {
```

my guess is, at some point, doctrine stopped giving us `null-null` as a special value? so the branch has been passing?

i'm not sure. the same "empty name" check now succeeds so we get `null` correctly